src/probe/deep.rs

```diff
@@ -330,6 +330,7 @@ impl DeepProbe {
       return Ok(());
     }
 
+    let mut frame_duration = 0.0;
```

Comment:
Si tu le definis la, on peut le passer en parametre partout ou on en a besoin, ou bien on le definit tout le temps dans la fonction de la detection.
Je trouve plus propre de l'avoir seulement dans la fonction de detection, mais ca veut dire qu'on le recalcule a chaque fois (pareil pour le frame_rate, ...)

Reply:
tu parles de quelle fonction de détection ici ?

Reply:
blackframes, dualmono, silence, black and silence, je pensais qu'il y en avait plus

Reply:
ok j'avais pas bien compris ton premier commentaire, c'est vrai que c'est pas très logique de l'avoir mis en paramètre juste pour le black_and_silence, je vais le déplacer

Reply:
Ah non en fait c'est qu'on a pas le context dans le black_and_silence, comme c'est une fonction qui utilise des StreamProbeResult déjà existant, c'est un peu dommage de lire à nouveau dans le fichier. Finalement passer les framerate, frame duration etc en paramètre c'est mieux je pense.